jobs/mysql_backup_job.go

```diff
@@ -154,25 +163,14 @@ func (j *MySQLBackupJob) Run(ctx context.Context, send Send) (rerr error) {
 			return wrapError(err)
 		}
 
-		defer func() {
-			err := xbcloudCmd.Wait()
-			if err == nil {
-				return
-			}
-
-			if rerr != nil {
-				rerr = errors.Wrapf(rerr, "xbcloud wait error: %s", err)
-			} else {
-				rerr = wrapError(err)
-			}
-		}()
-	}
-
-	if err := xtrabackupCmd.Start(); err != nil {
-		return wrapError(err)
+		if err := xbcloudCmd.Wait(); err != nil {
+			_ = xtrabackupStdout.Close()
```

Comment:
so, closing stdout is a key, maybe that change be enough?
I think better option would be to terminate xtrabackup instead of closing stdout.

Reply:
When I was testing it before changes. It would hang on both `Wait()`
With this changes without stdout close, only one hangs. With close, no one hangs.

Reply:
Don't we make xtrabackup fail by closing stdout? maybe it's better to terminate the process, since it's what we want to do